lib/index.js

```diff
@@ -107,28 +107,40 @@ api.set = async ({
   } else if(type === 'nonce') {
     // FIXME: may need to allow up to N tokens, one per `clientId` if
     // `clientId` is set
-
-    // if there is an existing nonce, delete it, and generate a new one
-    let existingToken;
+    const maxNonceCount = 5;
```

Comment:
We should make this an option in the bedrock config for this module -- and default it to `5` there.